src/main/app/src/app/zaken/zaken-mijn/zaken-mijn.component.ts

```diff
@@ -99,4 +103,40 @@ export class ZakenMijnComponent implements OnInit, AfterViewInit, OnDestroy {
         return Conditionals.isOverschreden(datum);
     }
 
+    private setColumns() {
+        if (this.werklijstData) {
+            const mapColumns: Map<string, ColumnPickerValue> = new Map(JSON.parse(this.werklijstData.columns));
+            this.dataSource.initColumns(mapColumns);
+        } else {
+            this.dataSource.initColumns(this.initialColumns());
+        }
+    }
+
+    initialColumns(): Map<string, ColumnPickerValue> {
+        return new Map([
+            ['identificatie', ColumnPickerValue.VISIBLE],
+            ['status', ColumnPickerValue.VISIBLE],
+            ['zaaktype', ColumnPickerValue.VISIBLE],
+            ['groep', ColumnPickerValue.VISIBLE],
+            ['startdatum', ColumnPickerValue.VISIBLE],
+            ['einddatum', ColumnPickerValue.HIDDEN],
+            ['einddatumGepland', ColumnPickerValue.HIDDEN],
+            ['aanvrager', ColumnPickerValue.VISIBLE],
+            ['uiterlijkeEinddatumAfdoening', ColumnPickerValue.HIDDEN],
+            ['toelichting', ColumnPickerValue.HIDDEN],
+            ['url', ColumnPickerValue.STICKY]
+        ]);
+    }
+
+    resetSearchCriteria() {
+        if (this.werklijstData) {
+            this.dataSource.filters = {};
+            this.dataSource.initColumns(this.initialColumns());
+            this.paginator.pageIndex = 0;
+            this.paginator.pageSize = 25;
+            this.sort.active = '';
```

Comment:
Idem

Reply:
Sort.active en sort.direction zijn uiteindelijk beide strings. Er is geen methode om ze te resetten/clearen.